sound/soc/sof/intel/hda-stream.c

```diff
@@ -657,6 +658,9 @@ int hda_dsp_stream_init(struct snd_sof_dev *sdev)
 		list_add_tail(&hstream->list, &bus->stream_list);
 	}
 
+	/* store the total playback + capture stream number */
+	sof_hda->stream_total = num_total;
```

Comment:
why is this needed? You don't use this information for anything in this patch set?

Reply:
it is used in the consequent stream interrupt refining one, to configure the correct bit mask for interrupt.

Reply:
somehow your commits are not in the right order in GitHub? It makes the reviews really hard...